lib/Room.rb

```diff
@@ -0,0 +1,30 @@
+module Hotel
+  class Room
+
+    attr_reader :room_num
+    attr_accessor :rate
+
+    STANDARD_RATE = 200
+    @@rooms = []
+
+    def initialize(room_num)
+      @room_num = room_num
+      @rate = STANDARD_RATE
+      collect_instance
+    end
+
+    def collect_instance
+      @@rooms.push(self)
+    end
+
+    def self.all
+      return @@rooms
+    end
+
+    def self.update_price(room_num, new_rate)
```

Comment:
Why not provide a way in the `initialize` method to start off a room with an optional parameter for price.